pkg/v1/config/clientconfig.go

```diff
@@ -200,6 +202,16 @@ func addEdition(c *configv1alpha1.ClientConfig, edition configv1alpha1.EditionSe
 	c.ClientOptions.CLI.Edition = edition
 }
 
+func addCompatabilityFile(c *configv1alpha1.ClientConfig, comapatibilityFileLocation string) {
```

Comment:
This function is called in exactly one spot - could we inline it to `addDefaultEditonIfMissing`?

Reply:
I did it to keep the structure consistent with:

https://github.com/vmware-tanzu/tanzu-framework/blob/caabeefb01e299e3b03dd8809925a595086eea53/pkg/v1/config/clientconfig.go#L182-L203

The compiler should be smart enough to inline it, so I think this is fine.

LMK if you or anyone feels passionately otherwise!